modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -40,6 +40,7 @@ image::op-pipeline-builder-task-details.png[]
 .. In the *Output Resources* -> *Image* section, click the *Select Resources* list, and select *Img*.
 .. If required, in the *Parameters* section, add more parameters to the default ones, by using the `$(params.<param-name>)` syntax.
 . Similarly, add an input resource for the *openshift-client* Task.
-. Click *Create* to create the Pipeline. You are redirected to the *Pipeline Details* page that displays the details of the created Pipeline. You can now use the *Action* button to start the Pipeline.
+. Click *Create* to create the Pipeline. You are redirected to the *Pipeline Details* page that displays the details of the created Pipeline.
+. Click the *Actions* drop-down menu, and then click Start to start the Pipeline.
```

Comment:
This should state "and then click **Start** to start". I.e. the first "Start" needs to be in bold because it is a UI element. Start is in bold [in the original PR](https://github.com/openshift/openshift-docs/pull/29687/files#diff-2b69c47532e944f8e5c6873559cd2d81f62745910ddedb28769431b0a199370dR58).

Reply:
Done. Thanks a lot!